test/types/Integral/integralvariable.bad

```diff
@@ -0,0 +1,3 @@
+internal error: CAL0056 chpl Version 1.12.0  Internal errors indicate a bug in the Chapel compiler ("It's us, not you"), and we're sorry
+for the hassle.We would appreciate your reporting this bug --  please see http://chapel.cray.com/bugs.html for instructions.  
+In the meantime, the filename + line number above may be useful in working around the issue.
```

Comment:
The spacing of this does not look quite right to me - an example of a .bad with the same format is test/classes/constructors/assign-param-segfault.bad - if you make the line output similar to that, you'll be in better shape.